NSS-Spontaneous-Spoonbills/Sprint2
# Db templates

## Link to Ticket

#10 #9 

Closes #
none
## Description of Proposed Changes
modified Training_Model.py, urls.py(WorkforceManagement), Training_Detail.html, Training_List.html, Training_View.py. corrected minor naming issues in the mentioned files. 

## Steps to Test

Outline the steps to test 

```
git fetch --all
git checkout db_templates
```
remove migrations and db.sqlite
in terminal 
```
python manage.py makemigrations WorkforceManagement
python manage.py migrate
```
in DB browser make entry for workforcemanagement training programs

in browser make sure your entry is viewable
## Impacted Areas in Application

user should now be able to view training programs
